react-native-libraries.json

```diff
@@ -10288,5 +10288,14 @@
       "https://github.com/neurio/react-native-local-network-permission/tree/main/example"
     ],
     "ios": true
+  },
+  {
+    "githubUrl": "https://github.com/flixyudh/react-native-flix-snackbar",
+    "npmPkg": "react-native-flix-snackbar",
```

Comment:
Hello @flixyudh, thank you for adding new entry to the directory! 👍 

```suggestion
```

Small nit, `npmPkg` can be skip in this case. 🙂

Reply:
got it @Simek